diff --git a/node_modules/@angular-devkit/build-angular/src/webpack/configs/common.js b/node_modules/@angular-devkit/build-angular/src/webpack/configs/common.js
index f4dd8d7..e7847c9 100644
--- a/node_modules/@angular-devkit/build-angular/src/webpack/configs/common.js
+++ b/node_modules/@angular-devkit/build-angular/src/webpack/configs/common.js
@@ -292,27 +292,27 @@ function getCommonConfig(wco) {
             exclude: globalScriptsNames,
             terserOptions,
         }),
-        // Script bundles are fully optimized here in one step since they are never downleveled.
-        // They are shared between ES2015 & ES5 outputs so must support ES5.
-        new TerserPlugin({
-            sourceMap: scriptsSourceMap,
-            parallel: utils_1.maxWorkers,
-            cache: !environment_options_1.cachingDisabled && cache_path_1.findCachePath('terser-webpack'),
-            extractComments: false,
-            include: globalScriptsNames,
-            terserOptions: {
-                ...terserOptions,
-                compress: environment_options_1.allowMinify && {
-                    ...terserOptions.compress,
-                    ecma: 5,
+            // Script bundles are fully optimized here in one step since they are never downleveled.
+            // They are shared between ES2015 & ES5 outputs so must support ES5.
+            new TerserPlugin({
+                sourceMap: scriptsSourceMap,
+                parallel: utils_1.maxWorkers,
+                cache: !environment_options_1.cachingDisabled && cache_path_1.findCachePath('terser-webpack'),
+                extractComments: false,
+                include: globalScriptsNames,
+                terserOptions: {
+                    ...terserOptions,
+                    compress: environment_options_1.allowMinify && {
+                        ...terserOptions.compress,
+                        ecma: 5,
+                    },
+                    output: {
+                        ...terserOptions.output,
+                        ecma: 5,
+                    },
+                    mangle: environment_options_1.allowMangle && platform !== 'server',
                 },
-                output: {
-                    ...terserOptions.output,
-                    ecma: 5,
-                },
-                mangle: environment_options_1.allowMangle && platform !== 'server',
-            },
-        }));
+            }));
     }
     return {
         mode: scriptsOptimization || stylesOptimization.minify ? 'production' : 'development',
@@ -374,6 +374,7 @@ function getCommonConfig(wco) {
                 },
                 {
                     test: /\.[cm]?js$|\.tsx?$/,
+                    resolve: { fullySpecified: false },
                     exclude: [/[\/\\](?:core-js|\@babel|tslib|web-animations-js)[\/\\]/],
                     use: [
                         {
