diff --git a/node_modules/@cds/core/password/password.element.d.ts b/node_modules/@cds/core/password/password.element.d.ts
index dc85edf..b9ec95f 100644
--- a/node_modules/@cds/core/password/password.element.d.ts
+++ b/node_modules/@cds/core/password/password.element.d.ts
@@ -34,7 +34,6 @@ export declare class CdsPassword extends CdsControl {
         hideButtonAriaLabel: string;
     };
     constructor();
-    private get ariaLabel();
     protected get suffixDefaultTemplate(): import("lit-html").TemplateResult<1>;
     static get styles(): import("lit").CSSResultGroup[];
     togglePasswordVisibility(): void;
